client/main.js

```diff
@@ -97,7 +97,11 @@ main = (function(main, global) {
     });
 
 
-    $('#nav-chat-btn').on('click', toggleMainChat);
+    $('#nav-chat-btn').on('click', function(event) {
+      if (!event.ctrlKey && !event.metaKey) {
```

Comment:
Awesome, @george-stepanek if you can make that change than this PR can be merged.